app/src/main/java/org/mozilla/fenix/compose/LightDarkPreview.kt

```diff
@@ -0,0 +1,8 @@
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.ui.tooling.preview.Preview
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
+annotation class LightDarkPreview
```

Comment:
🚗 I'm not super familiar with annotation classes, but I assume `LightDarkPreview` doesn't expose the parameters available to `@Preview` by some transitive magic or anything. Should we consider adding those same parameters here? Is there a way to pass them to both of the super classes? 

I'm just concerned that we could lose some flexibility in the our preview options if this becomes a common usage.

Reply:
this is a good point, I'll look into the options for `@Preview`

Reply:
hi @MatthewTighe 

after some further investigation, I have discovered that we mainly use only three fields for the following scenarios:
1) locale : this is used for Right to Left and Left to Right display previews. 
2) uiMode: this is used for Night Mode on and off 
3) name: used for labeling  

This new multi-annotation preview class would mainly be used to replace option 2 as it is the most common, used just with the night mode on and off.

  I also discovered that I cannot pass the parameters up to the multi-annotations like in this example: 
```
@Preview(name = name, uiMode = Configuration.UI_MODE_NIGHT_YES)
@Preview(name = name, uiMode = Configuration.UI_MODE_NIGHT_NO)
annotation class LightDarkPreview(
    val name: String = "",
    val group: String = "",
    @IntRange(from = 1) val apiLevel: Int = -1,
    // TODO(mount): Make this Dp when they are inline classes
    val widthDp: Int = -1,
    // TODO(mount): Make this Dp when they are inline classes
    val heightDp: Int = -1,
    val locale: String = "",
    @FloatRange(from = 0.01) val fontScale: Float = 1f,
    val showSystemUi: Boolean = false,
    val showBackground: Boolean = false,
    val backgroundColor: Long = 0,
    @UiMode val uiMode: Int = 0,
    @Device val device: String = Devices.DEFAULT
)
```
 my proposal is that when more annotation fields are needed, use the original @Preview() annotation with the fields filled in while keeping this multi-annotation preview class for just the Night Mode on and off scenario.

Reply:
That sounds good to me! Shame the parameters can't be shared. Seems like it would be super helpful to have the name propagate, for instance.